content/v2/registrar.markdown

```diff
@@ -198,6 +198,7 @@ Name | Type | Description
 -----|------|------------
 `registrant_id` | `integer` | **Required**. The ID of an existing [contact](/v2/contacts/#contact-attributes) in your account.
 `whois_privacy` | `bool` | Set to true will attempt to purchase/enable the whois privacy as part of the registration. An extra cost may apply. Default: `false`.
+`linked_provider` | `string` | When set to the nickname of a linked provider, registration of the domain will go through that provider. Otherwise, registration of the domain will go through DNSimple.
```

Comment:
How are provider created? Do you have a screenshot? Is the field called "nickname"? We use more often terms like label, name, or else.

Reply:
The field on creation asks for a Nickname. Once created the nickname is shown under the Linked Providers section:

<img width="786" alt="Screenshot 2023-09-28 at 11 34 41 AM" src="https://github.com/dnsimple/dnsimple-developer/assets/47795400/67287b27-d777-4bae-a02c-4e35dbc2dd8c">

- I think `provider_nickname` might be more explicit for the name
- Is it possible to use the same nickname for different providers? What If I want a `default` nickname for each linked provider?

From a user perspective, If I have multiple providers each with multiple accounts linked, I am heavily relying on nicknames to determine which provider/account combination I am using. 

From an operational/readability perspective:

```
provider : "route53"
provider_nickname : "default"
```

OR

```
linked_provider: "route53:default"
```

Reply:
It’s rather uncommon indeed. I like it because it makes the distinction to name clear. As the provider name is GoDaddy and we let customers assign an account-unique “nickname”. I think another maybe more “common” name would be alias.

Reply:
Currently, nicknames are unique within an account, so I do not see an issue using them as an identifier.

Reply:
What should we use for the param name? I think `provider_nickname` might be more clear and is developer-friendly with the conventions we are used to.

Reply:
Are you considering adding additional validation on the characters allowed to be used in the nickname?

Reply:
We have not considered adding any validations except the ones that we have right now. The only validation we have right now is that we require the nickname to be present.